modules/sql-engine/src/integrationTest/sql/update/test_null_update.test

```diff
@@ -0,0 +1,29 @@
+# name: test/sql/update/test_null_update.test
+# description: Test standard update behavior with NULLs
+# group: [update]
+
+# create a table
+statement ok
+CREATE TABLE test (a INTEGER);
+
+statement ok
+INSERT INTO test VALUES (1), (2), (3), (NULL)
+
+query I
+SELECT * FROM test ORDER BY a
+----
+1
+2
+3
+NULL
+
+statement ok
+UPDATE test SET a=NULL WHERE a=2
```

Comment:
should we check also ``UPDATE test SET=4 WHERE a IS NULL`` ?

Reply:
Thanks. Added a test case for that.